doc/Language/functions.rakudoc

```diff
@@ -1018,6 +1018,28 @@ dispatch, or it could be the inner routine from a C<wrap>.
 Fortunately, we have a series of re-dispatching tools that help us to make
 it easy.
 
+=head2 The Next Tools
+
+=begin table
+
+                    | Returns | Parameters | Referee
+    ================+=========+============+========
+    nextsame        | No      | Reuse      | next
+    nextwith        | No      | Specify    | next
+    callsame        | Yes     | Reuse      | next
+    callwith        | Yes     | Specify    | next
+    nextcallee      | Yes     | None       | next
+    samewith        | Yes     | Specify    | self
+    nextcallee's rv | Yes     | Specify    | preset
+
+=end table
+
+From the above we learn:
+
+=item C<with> always means that the parameters need to be specified
+=item C<same> at word end means the parameters of the referee will automatically be reused
```

Comment:
can we use the same languange to describe both of these? (pease consider "at the end of the name")

Reply:
I've rephrased this section.  However, I want to highlight that what I'm doing here is highlighting gotchas.  I'm not sure using the same language would support that goal, but am happy to take suggestions along that line.